corehq/apps/reports/models.py

```diff
@@ -268,7 +269,11 @@ def _dispatcher(self):
             if dispatcher.prefix == self.report_type:
                 return dispatcher()
 
-        raise Exception("Unknown dispatcher: %s" % self.report_type)
+        logging.error("This saved-report (id: %s) is unknown (report_type: %s). (Might be a legacy report)" % (
```

Comment:
String formatting is happening outside the logging call (you're using the `%` operator). It's more efficient to pass string formatting arguments into the logging call.